src/unity/python/turicreate/toolkits/drawing_recognition/drawing_recognition.py

```diff
@@ -30,8 +33,8 @@ def _raise_error_if_not_drawing_classifier_input_sframe(dataset, feature):
             + " where each stroke is a list of points and" 
             + " each point is stored as a dictionary")
 
-def create(input_dataset, annotations=None, num_epochs=100, feature="bitmap", 
-           target="label", model=None, classes=None, batch_size=256, 
+def create(input_dataset, num_epochs=100, feature="bitmap", 
```

Comment:
Does `num_epochs` not depend in some way on whether we're using a pretrained model?

Reply:
Not at the moment. I was thinking that could come in as part of polish or v2.